Atomic/scan.py

```diff
@@ -12,14 +12,15 @@ def cli(subparser):
     # atomic scan
     scanners = util.get_scanners()
     scanp = subparser.add_parser(
-        "scan", help=_("scan an image or container for CVEs"),
-        epilog="atomic scan <input> scans a container or image for CVEs")
+        "scan", help=_("scan an image or container for CVEs or configuration compliance"),
+        epilog="atomic scan <input> scans a container or image for CVEs or configuration compliance")
     scanp.set_defaults(_class=Scan, func='scan')
     scan_group = scanp.add_mutually_exclusive_group()
     scanp.add_argument("scan_targets", nargs='*', help=_("container image"))
     scanp.add_argument("--scanner", choices=[x['scanner_name'] for x in scanners], default=None, help=_("define the intended scanner"))
     scanp.add_argument("--scan_type", default=None, help=_("define the intended scan type"))
     scanp.add_argument("--list", action='store_true', default=False, help=_("List available scanners"))
+    scanp.add_argument("--profile_id", default=None, help=_("Select a SCAP Security Guide profile"))
```

Comment:
I will let @baude comment.    Is this nothing more then an args that gets passed to the scanner?  I am looking for a way to make this not OpenScap scanner specific and allow other scanners to pass in  additional args to their scanners.